components/service/glean/src/test/java/mozilla/components/service/glean/debug/GleanDebugActivityTest.kt

```diff
@@ -140,4 +140,102 @@ class GleanDebugActivityTest {
 
         server.shutdown()
     }
+
+    @Test
+    fun `pings are tagged using tagPings`() {
+        val server = MockWebServer()
+        server.enqueue(MockResponse().setBody("OK"))
+
+        resetGlean(config = Glean.configuration.copy(
+            serverEndpoint = "http://" + server.hostName + ":" + server.port
```

Comment:
`"http"`?
That means plaintext transmissions.
Are we not using `https` in Glean everywhere?

Reply:
Please check with akomar if `https` is availabe on this endpoint.

Reply:
We use `https` in glean everywhere in practice.  This is to communicate with the `MockWebServer` that gets fired up for this specific test.  It looks like it [might be possible](http://square.github.io/okhttp/3.x/mockwebserver/index.html?okhttp3/mockwebserver/MockWebServer.html) to get that server to speak `https` and we could look into doing that.

Are you suggesting the glean should enforce that the `serverEndpoint` is always https even when it's set to a custom value?

Reply:
Whoops, I think you misunderstand what's going on here.  This is creating a local server in a unit test in order to capture the pings and make sure they look good upon receipt.  We don't want unit test pings to get to the pipeline.

Reply:
@travis79 can you please file a bug for enforcing *HTTPS* outside of tests?

Reply:
Bug filed [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1538689)